spec/models/post_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.create(author_id: 1, Title: 'Hello', Text: 'This my first post') }
+  before { subject.save }
+
+  it 'title should be not blank' do
+    subject.Title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should be not blank' do
+    subject.Title = 't' * 251
+    expect(subject).to_not be_valid
+  end
+  it 'CommentsCounter must be an integer ' do
+    subject.Comments_Counter = 'text'
+    expect(subject).to_not be_valid
+  end
+  it 'CommentsCounter must be greater than or equal to zero ' do
+    subject.Comments_Counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'LikesCounter must be an integer greater than or equal to zero. ' do
+    subject.Likes_Counter = 'word'
+    expect(subject).to_not be_valid
+  end
+  it 'LikesCounter must be an integer greater than or equal to zero. ' do
+    subject.Likes_Counter = -1
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] Kindly add a test for post counter method. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods)